### DISCUSSION TITLE
Fix reporting on Vercel platform

### DISCUSSION THREAD

**Discussion by qendrimkryeziu** [vercel/community#5243] [Bug Report]
### Page to Investigate

https://vercel.com/login

### Steps to Reproduce

Hi Support Vercel,

I want to report a fix on the Vercel platform. I have a problem accessing the platform, when I try to log in I get this error message: There is no Vercel account associated with this email address. Sign up?

When I try to sign up, I get this error again: The email address already exists. Login?

whereas when I try to connect with a github account, I get this message: There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account. Login with Email.

My report is how is it possible that the platform tells me that you are registered in the sign up, but in the login it says that you do not have a user?

My account email is: qendrim.kryeziu@tetbit.com

For more, you also have the screenshots.

Thank you!

<img width="1792" alt="Screenshot 2023-12-12 at 11 28 20" src="https://github.com/vercel/community/assets/80457722/30f13f55-b0a4-4761-9089-da5a2cf8d643">
<img width="1792" alt="Screenshot 2023-12-12 at 11 29 34" src="https://github.com/vercel/community/assets/80457722/2af0aa56-a90c-4c16-8630-e06af3cabced">
<img width="1792" alt="Screenshot 2023-12-12 at 11 30 40" src="https://github.com/vercel/community/assets/80457722/67ebc1a8-b222-42be-90cd-de9567427094">

**Comment by amyegan** [comment]
Hey @qendrimkryeziu. Thanks for reporting this. If you try logging in with that email again, but remove the `.` from the username, does that succeed?

**Reply by qendrimkryeziu** [reply]
Hey @amyegan,  If I try without . I need to be verified but I can't because my email is with . and it comes to my inbox or my space

<img width="1792" alt="Screenshot 2023-12-21 at 15 47 38" src="https://github.com/vercel/community/assets/80457722/d0c0919f-be5a-462f-88ba-cc8d727d4e24">
